www-apps/otrs/otrs-5.0.15.ebuild

```diff
@@ -50,7 +50,7 @@ RDEPEND="dev-perl/Apache-Reload
 		!=dev-perl/SOAP-Lite-0.712 )
 	"
 
-OTRS_HOME="/var/lib/otrs"
+OTRS_HOME="${EROOT}/var/lib/otrs"
```

Comment:
*EROOT* ends with */* so I would suggest to use `${EROOT%/}` so that *OTRS_HOME* doesn't end up with a value like `//var/lib/otrs` (pure optical...).